ci/build.sh

```diff
@@ -5,7 +5,13 @@ PATH="$HOME/depot_tools:$PATH"
 
 cd ..
 
+# Build the dart UI files
 flutter/tools/gn --unoptimized
 ninja -C out/host_debug_unopt generate_dart_ui
+
+# Analyze the dart UI
 flutter/ci/analyze.sh
 flutter/ci/licenses.sh
+
+ # Build full host engine for upcoming tests
+ninja -C out/host_debug_unopt
```

Comment:
Question: how does this affect presubmit time? If it's significantly slower, @liyuqian probably has ideas.

Reply:
It raises it for sure. I'll wait on @liyuqian to look this over.

Reply:
Remove this line as you've already moved run_tests.sh to the compile_and_test_host task. As we discussed offline, compiling host needs a lot of CPU power so we want to do that on our own GKE cluster instead of on default Cirrus cluster.